source/speech.py

```diff
@@ -1405,6 +1405,8 @@ def getFormatFieldSpeech(attrs,attrsCache=None,formatConfig=None,unit=None,extra
 		linePrefix=attrs.get("line-prefix")
 		if linePrefix:
 			textList.append(linePrefix)
+	breakpoint=attrs.get("breakpoint")
+	if breakpoint: textList.append(breakpoint)

```

Comment:
Fixed most of the issues except the breakpoint format field attribute.

 

I don’t know how to make NVDA speak break points without it?

 

From: James Teh [mailto:notifications@github.com] 
Sent: Tuesday, March 08, 2016 9:17 AM
To: nvaccess/nvda
Cc: dineshkaushal
Subject: Re: [nvda] In t2233 new (#5598)

 

In source/speech.py <https://github.com/nvaccess/nvda/pull/5598#discussion_r55313010> :

> @@ -1405,6 +1405,8 @@ def getFormatFieldSpeech(attrs,attrsCache=None,formatConfig=None,unit=None,extra
>              linePrefix=attrs.get("line-prefix")
>               if linePrefix:
>                      textList.append(linePrefix)
> +     breakpoint=attrs.get("breakpoint")
> +     if breakpoint: textList.append(breakpoint)

@MichaelDCurran <https://github.com/MichaelDCurran> , thoughts on adding a breakpoint format field attribute? It seems almost app specific, but I guess it does seem odd abusing line-prefix. Is there any more generic concept here?

—
Reply to this email directly or view it on GitHub <https://github.com/nvaccess/nvda/pull/5598/files#r55313010> .  <https://github.com/notifications/beacon/AE08v4t1Qra5UL0boUWXxeW3S_uB_xcWks5prPEfgaJpZM4Gyk21.gif>